server/src/domain/job/job.constants.ts

```diff
@@ -44,6 +45,11 @@ export enum JobName {
   STORAGE_TEMPLATE_MIGRATION_SINGLE = 'storage-template-migration-single',
   SYSTEM_CONFIG_CHANGE = 'system-config-change',
 
+  // migration
+  QUEUE_MIGRATION = 'queue-migration',
+  ASSET_MIGRATION = 'asset-migration',
+  FACE_THUMBNAIL_MIGRATION = 'face-thumbnail-migration',
```

Comment:
```suggestion
  MIGRATE_PERSON = 'migrate-person',
```

We're passing the entity of a person. Who knows, maybe in the future they will have videos too. I think keep it general "migrate this person" seems better, and matches migrating an asset, which is made up of multiple files that are looked at.